src/authorization.ts

```diff
@@ -20,44 +20,64 @@ export interface BulkDataAuth {
     importResources?: string[];
 }
 
+/**
+ * Contains information from incoming http request. As well as
+ * additional context information such use case specific auth handlers or
+ * operational information specific to this request.
+ */
+export interface RequestContext {
+    method: string;

```

Comment:
Is this URL VERB? i.e. POST, GET, etc? or is this the operation? 'create', 'update', etc

Reply:
I'd say this is the HTTP verb, like POST, GET etc.

Reply:
I'd suggest changing the name to verb and making it stricter here seems to be a list of them all: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

```suggestion
    verb: 'CONNECT' |  'DELETE' | 'GET'| 'HEAD'| 'OPTIONS'|'PATCH'|'POST'|'PUT'|'TRACE';

```

Reply:
Great suggestion! I'll change that.